release-notes.md

```diff
@@ -1,3 +1,17 @@
+# Release 1.8.2
+
+## What's New
+* [#317](https://github.com/openziti/desktop-edge-win/issues/317) command line list function to fetch identities and services
```

Comment:
is there any doc about the usage that could be added to the release notes or a markdown? or can someone just running it expect to get back reasonable help?

Reply:
ziti-tunnel list -h will display the help message